src/modules/0.0.22/aws_cloudwatch/index.ts

```diff
@@ -58,23 +58,25 @@ class LogGroupMapper extends MapperBase<LogGroup> {
     },
     read: async (ctx: Context, id?: string) => {
       const enabledRegions = (await ctx.getEnabledAwsRegions()) as string[];
-      if (id) {
-        for (const region of enabledRegions) {
+      if (!!id) {
+        const [groupName, region] = id.split('|');
```

Comment:
maybe this should be a stateless/static getter function in the cloud id service?

Reply:
Agreed. We should have it as a built-in auto-defined method like `entityId` that returns an object of field -> value pieces. This is completely doable since the fields are stored in an array in the same order as the `id.split('|')` output [here](https://github.com/iasql/iasql-engine/blob/main/src/services/cloud-id.ts#L1).

Perhaps it could be called `this.idFields(id: string)`?

Reply:
agree! I can add that in this PR